frontend/test/metabase/scenarios/alert/alert_auth.cy.spec.js

```diff
@@ -0,0 +1,108 @@
+import {
+  restore,
+  signInAsAdmin,
+  signInAsNormalUser,
+  setupLocalHostEmail,
+  createBasicAlert,
+} from "../../../__support__/cypress";
+// Port from alert.e2e.spec.js
+
+describe("scenarios > alert > auth for alerts", () => {
+  before(() => {
+    restore();
+    signInAsAdmin();
+
+    // Setup email
+    // NOTE: Must run `python -m smtpd -n -c DebuggingServer localhost:1025` before these tests
```

Comment:
We're going to run these tests in CI though, right? Is there a separate SMTP server set up there?

Reply:
Not yet, at least not that I'm aware of- I should have these tests skipped until that's set up.